examples/kafka-test/asyncapi.yaml

```diff
@@ -0,0 +1,18 @@
+asyncapi: 2.0.0
+info:
+  title: Kafka test
+  version: '1'
+servers:
+  mykafka:
+    url: kafka://pkc-6ojv2.us-west4.gcp.confluent.cloud:9092
+    protocol: kafka-secure
+channels:
+  test:
+    publish:
+      operationId: onTest
+      message:
+        payload:
+          type: object
+          properties:
+            test:
+              type: string
```

Comment:
Your `onTest` function is sending a message to the topic `produce` so that should be defined in the AsyncAPI file:

```suggestion
        $ref: '#/components/messages/testMessage'
  produce:
    subscribe:
      message:
        $ref: '#/components/messages/testMessage'
components:
  messages:
    testMessage:
      payload:
        type: object
        properties:
          test:
            type: string
```

Otherwise, you'll get the following error:

<img width="665" alt="Captura de Pantalla 2022-12-22 a las 19 18 53" src="https://user-images.githubusercontent.com/242119/209200128-ca7a811c-c6d3-4639-847f-05351ae892a6.png">